docs/tutorial.md

```diff
@@ -166,7 +166,7 @@ Such mutations would *not* be legal, on the other hand, if we set the tskit sequ
 since the position 99 would be outside of the interval from 0 to 99.
 Said another way, if SLiM's final position is 99, the total sequence length is 100,
 and so we need to set the end of the genome to 100.
-The upshot is that we need to use SLiM's last position plus one - i.e., 
+The upshot is that we need to use SLiM's last position plus one - i.e.,
```

Comment:
sorry about all the noise here - I did a search-replace for spaces at the ends of lines